### PR TITLE
Telemetry API: custom metrics

### PR DESCRIPTION
**Pull request by zirain**
cc @kyessenov @lei-tang

### CODE REVIEW

File: content/en/docs/tasks/observability/metrics/telemetry-api/index.md [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+  meshConfig:
+    defaultConfig:
+      extraStatTags:
+        - source_x
+        - destination_x
+  values:
+    telemetry:
+      enabled: true
+      v2:
+        enabled: false
```
Comment (lei-tang) [inline content/en/docs/tasks/observability/metrics/telemetry-api/index.md:34]: Thanks for the PR!  To help readers understand the configuration here, in the paragraph above, can you add an explanation of why telemetry.v2.enabled is set as false?

File: content/en/docs/tasks/observability/metrics/telemetry-api/index.md [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+        - providers:
+            - name: prometheus
+          overrides:
+            - disabled: true
+              match:
+                mode: SERVER
+                metric: REQUEST_COUNT
+    {{< /text >}}
+
+## Custom metric reporting interval
```
[the remark this answers is not in the document]
Comment (zirain) [reply content/en/docs/tasks/observability/metrics/telemetry-api/index.md:167]: ok, I will revert this part first.

File: content/en/docs/tasks/observability/metrics/telemetry-api/index.md [hunk 3]
```diff
@@ -0,0 +1,165 @@
+---
+title: Customizing Istio Metrics with Telemetry API
+description: This task shows you how to customize the Istio metrics with Telemetry API.
+weight: 10
```
Comment (ericvn) [inline content/en/docs/tasks/observability/metrics/telemetry-api/index.md:4]: This weight will place this item as first on the page (https://deploy-preview-12916--preliminary-istio.netlify.app/latest/docs/tasks/observability/metrics/). Maybe you want it after the Customizing Istio Metrics, so then would be 26?

File: content/en/docs/tasks/observability/metrics/telemetry-api/index.md [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+keywords: [telemetry,metrics,customize]
+owner: istio/wg-policies-and-telemetry-maintainers
+test: no
+---
+
+Telemetry API has been in Istio as a first-class API for quite sometime now.
+Previously users had to configure metrics in the `telemetry` section of Istio configuration,
```
Comment (ericvn) [inline content/en/docs/tasks/observability/metrics/telemetry-api/index.md:11]:
```suggestion
Previously, users had to configure metrics in the `telemetry` section of the Istio configuration.
```

File: content/en/docs/tasks/observability/metrics/telemetry-api/index.md [hunk 5]
```diff
@@ -0,0 +1,165 @@
...
+This task shows you how to customize the metrics that Istio generates with Telemetry API.
+
+## Before you begin
+
+[Install Istio](/docs/setup/) in your cluster and deploy an application.
+Telemetry API can not work together with the `EnvoyFilter` way, for more details please checkout [this](https://github.com/istio/istio/issues/39772).
```
Comment (ericvn) [inline content/en/docs/tasks/observability/metrics/telemetry-api/index.md:17]:
```suggestion
Telemetry API can not work together with `EnvoyFilter`. For more details please checkout this [issue](https://github.com/istio/istio/issues/39772).
```

File: content/en/docs/tasks/observability/metrics/telemetry-api/index.md [hunk 6]
```diff
@@ -0,0 +1,165 @@
...
+From `1.18`, stats `EnvoyFilter` will not be installed by default.
```
Comment (ericvn) [inline content/en/docs/tasks/observability/metrics/telemetry-api/index.md:18]:
```suggestion
Starting with Istio version `1.18`, the stats `EnvoyFilter` will not be installed by default.
```

File: content/en/docs/tasks/observability/metrics/telemetry-api/index.md [hunk 7]
```diff
@@ -0,0 +1,165 @@
...
+For version before `1.18`, you should install with following configuration in `IstioOperator` configuration:
```
Comment (ericvn) [inline content/en/docs/tasks/observability/metrics/telemetry-api/index.md:19]:
```suggestion
For versions of Istio before `1.18`, you should install with following `IstioOperator` configuration:
```

**Review by ericvn — COMMENTED** [review]
Also, is there a way to verify that all the yamls are correct?

**Comment by zirain** [comment]
> Also, is there a way to verify that all the yamls are correct?

I manually test this yaml, will add tests later.

File: content/en/docs/tasks/observability/metrics/telemetry-api/index.md (hunk 3 above)
Comment (zirain) [reply content/en/docs/tasks/observability/metrics/telemetry-api/index.md:4]: I want to put this page first, cause Telemetry API is the recommended way.

**Pull request merged**